app/models/subscription.rb

```diff
@@ -154,16 +161,16 @@ def limit_by_location(vacancies, location, radius_in_miles)
     end
     # rubocop:enable Metrics/AbcSize
 
-    def handle_location(scope, criteria)
+    def handle_location(vacancies, criteria)
```

Comment:
This is just a cognitive refactor. We're not dealing with a `scope` anymore, but an array of vacancies in memory.